File: src/main/java/lec15/Lec15Main4.java
package lec15;

import java.util.HashMap;
import java.util.Map;

public class Lec15Main4 {

    public static void main(String[] args) {
        // JDK 8까지
        HashMap<Integer, String> oldMap = new HashMap<>();
        oldMap.put(1, "MONDAY");
        oldMap.put(2, "TUESDAY");

        // JDK 9부터
        Map.of(1, "MONDAT", 2, "TUESDAY");
    }
}

File: src/main/kotlin/lec15/Lec15Main4.kt
package lec15

fun main() {

    val oldMap = mutableMapOf<Int, String>()
    oldMap[1] = "MONDAY"
    oldMap[2] = "TUESDAY"

    mapOf(1 to "MONDAY", 2 to "TUESDAY")

    for (key in oldMap.keys) {
        println(key)
        println(oldMap[key])
    }

    for ((key, value) in oldMap.entries) {
        println(key)
        println(value)
    }
}
